addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -1284,6 +1284,35 @@ QUnit.test('basic rendering of message', async function (assert) {
     );
 });
 
+QUnit.test('should not be able to reply to temporary/transient messages', async function (assert) {
+    assert.expect(1);
+
+    this.data['mail.channel'].records.push({ id: 20 });
+    await this.start({
+        discuss: {
+            params: {
+                default_active_id: 'mail.channel_20',
+            },
+        },
+    });
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "/");
+    });
+    await afterNextRender(() =>
+        document.querySelector('.o_ComposerSuggestion').click()
+    );
+    await afterNextRender(() =>
+        document.querySelector('.o_Composer_buttonSend').click()
+    );
+    await afterNextRender(() => document.querySelector('.o_Message').click());
```

Comment:
Put comment above to tell click on message to show actions on the transient message resulting from the "/who" command.